syntaxes/omt.tmLanguage.json

```diff
@@ -386,12 +386,29 @@
                     "patterns": [{ "include": "source.odt" }]
                 },
                 {
-                    "match": "(?:\\G|^) +(busyDisabled): *(\\w*)(.*)$",
+                    "match": "(?:\\G|^) +(busyDisabled|blockedIndication|pushmessageIndication): *(\\w*)(.*)$",
```

Comment:
`busyDisabled` is the only boolean property. `blockedIndication` and `pushmessageIndication` should be added to the query type properties on line 370